src/utils/webrtc/webrtc.js

```diff
@@ -935,10 +938,11 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		} else {
 			message = t('spreed', 'Error while accessing microphone & camera')
 			console.error('Error while accessing microphone & camera: ', error.message || error.name)
+			timeout = TOAST_DEFAULT_TIMEOUT
```

Comment:
I think `Error while accessing microphone & camera` is rather important one? should be permanent?

Reply:
I've separated the error in more detailed ones. In case the error is unknown, it will stay permanent.